### DISCUSSION TITLE
Other way to filter repositories than by selection in the app itself?

### DISCUSSION THREAD

**Discussion by beatngu13** [renovatebot/renovate#31466] [Request Help] [answered]
### What would you like help with?

I would like help with my configuration

### How are you running Renovate?

Mend Renovate hosted app on github.com

### Please tell us more about your question or problem

From the [docs](https://docs.renovatebot.com/getting-started/installing-onboarding/#hosted-githubcom-app):

>Renovate will ignore any repositories that don't have known package files, as well as any forks, so you can enable Renovate for all your repositories with no problems. That said, most people run Renovate on selected repositories. Unfortunately GitHub doesn't offer a "select all except X, Y, Z" option, so you must select each repository where you want Renovate to run.

Is there a other way to filter repositories than by selection in the app itself? Self-hosted Renovate has, e.g., [`autodiscoverFilter`](https://docs.renovatebot.com/self-hosted-configuration/#autodiscoverfilter) or [`repositories`](https://docs.renovatebot.com/self-hosted-configuration/#repositories). In a more advanced setup, teams usually don't have access to apps of the organization. Having the repositories as code would enable them to open PRs themselves, while the organization still benefits from the Mend-hosted variant (easier setup etc.).

**Comment by RahulGautamSingh** [comment]
You could add a `renovate.json` file with `enabled:false` in the repos you wish to ignore. Bit tedious, but s simple script should be able to do it if there are many repos.

**Reply by beatngu13** [reply]
Thank you! This is indeed a bit tedious. New repositories would need a `renovate.json` to opt-out, I would prefer an opt-in solution.

**Comment by rarkins** [comment, marked as answer]
You can install All repositories in the app, but then configure Renovate to be disabled or silent by default (i.e. no repository activity). Then this allows for repository admins to selectively turn on/up Renovate per-repository, such as initiating an onboarding PR. It's not the same as "repositories as code", but lets teams without org admin access fully control which repos are active in the app

**Reply by beatngu13** [reply]
By "disabled or silent by default" you mean enabling "Silent Mode" under settings in the Mend Developer Portal?

If we would do that, how could a repository admin "selectively turn on/up Renovate per-repository"? Via the Portal as well?

**Reply by rarkins** [reply]
Yes and Yes

**Reply by beatngu13** [reply]
We just tested it, and it works like a charm – thank you!
